.expeditor/config.yml

```diff
@@ -98,11 +98,34 @@ subscriptions:
         ignore_labels:
          - "Expeditor: Skip All"
          - "Expeditor: Skip Changelog"
+     - trigger_pipeline:omnibus/adhoc:
+        not_if: built_in:bump_version
+        ignore_labels:
+         - "Expeditor: Skip Omnibus"
+         - "Expeditor: Skip All"
      - trigger_pipeline:omnibus/release:
         only_if: built_in:bump_version
         ignore_labels:
          - "Expeditor: Skip Omnibus"
          - "Expeditor: Skip All"
+     - built_in:build_gem:
```

Comment:
Looks like we are failing Expeditor validation because we do not have a top-level rubygems section. We need to pull in the top-level `rubygems` section from the inspec-5 branch as well. It should have a config for inspec and inspec-bin.